packages/runtime/datastore/src/fluidHandle.ts

```diff
@@ -33,30 +31,6 @@ export class FluidObjectHandle<
 		return this.routeContext.isAttached;
 	}
 
-	/**
```

Comment:
This is great! Removing all this complexity from handle implementation is awesome. This is definitely a step in the right direction. Eventually, I think handles should not even be involved here. Well, we should either just keep `attachGraph` so that it can be called during attachment or we can expose the `routeContext` in some way and call `bind` directly on that.

Reply:
Yeah, after this change it's clear, handle just forwards to the route context. Once everyone's using the interface, we can switch up what actual object it is we're passing around for binding.